ui/dashboards/src/context/DashboardProvider/DashboardProvider.tsx

```diff
@@ -42,18 +42,6 @@ export function useEditMode() {
   return useDashboardStore(({ isEditMode, setEditMode }) => ({ isEditMode, setEditMode }));
 }
 
-export function useDashboard() {
```

Comment:
should we keep this for the export feature?

Reply:
I figure we should probably just build a "getDashboardJson" or something into the store for export (whenever we get around to it).